### PR TITLE
feat(forms): add `required` prop to Wizard.Step

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Wizard/Step/Step.tsx [hunk 1]
```diff
@@ -49,6 +56,9 @@ function Step(props: Props) {
     return null
   }
 
+  const fieldProps =
```
Comment (joakbjerk) [inline packages/dnb-eufemia/src/extensions/forms/Wizard/Step/Step.tsx:59]: Could this be added directly to the required attribute for the `FieldProps` wrapper below, or are we planning to add more props to the `fieldProps` variable later?

**Review by joakbjerk — APPROVED** [review]
Looks good 👍

File: packages/dnb-eufemia/src/extensions/forms/Wizard/Step/Step.tsx (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/Wizard/Step/Step.tsx:59]: yeah, the change is there that we add more.

**Pull request merged**